src/app/containers/ATIAnalytics/canonical/__snapshots__/index.test.jsx.snap

```diff
@@ -1,7 +1,53 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Canonical ATI Analytics should render a noscript image for non-JS users 1`] = `
-<noscript>
-  &lt;img height="1px" width="1px" alt="" style="position:absolute" src="https://foobar.com?key=value&amp;key2=value2&amp;x8=[simorgh-nojs]"/&gt;
-</noscript>
+initialize {
```

Comment:
Do we have any idea why this particular snapshot is so different to the baseline? (I know this is not part of your changes, but the unit tests fail without this). I checked this file in latest, and it looks like the before diff: https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/canonical/__snapshots__/index.test.jsx.snap